stan-native/src/main/java/org/ballerinalang/nats/streaming/consumer/StreamingListener.java

```diff
@@ -137,11 +116,11 @@ private void executeResource(String subject, Object[] args) {
                                                                           connectedUrl, subject);
             properties.put(ObservabilityConstants.KEY_OBSERVER_CONTEXT, observerContext);
             runtime.invokeMethodAsync(service, ON_MESSAGE_RESOURCE,
-                                      null, metadata, new DispatcherCallback(),
-                                      properties, args);
+                                      null, metadata, new DispatcherCallback(connectedUrl, subject),
+                                      properties, PredefinedTypes.TYPE_NULL, args);
```

Comment:
This return type should be taken from the remote function signature and passed here. 
Ref: https://github.com/ballerina-platform/module-ballerinax-nats/pull/183